jobs/haproxy/spec

```diff
@@ -128,15 +128,24 @@ properties:
           - 10.20.10.11
           backend_port: 80 # optional - sets backend port - otherwise defaults to `port`
           ssl: true        # optional - enables ssl, and uses the `ha_proxy.ssl_pem` provided key
+
   ha_proxy.resolvers:
     description: "List of DNS servers"
     example:
       resolvers:
-        private: 10.0.0.2
-        public: 8.8.8.8
+        - private: 10.0.0.2
+          public: 8.8.8.8
```

Comment:
Is this suppose to be an array containing a single two-item hash, or an array of two single-item hashes?